logstash-forwarder.go

```diff
@@ -79,14 +75,10 @@ func init() {
 	flag.BoolVar(&options.tailOnRotate, "tail", options.tailOnRotate, "always tail on log rotation -note: may skip entries ")
 	flag.BoolVar(&options.tailOnRotate, "t", options.tailOnRotate, "always tail on log rotation -note: may skip entries ")
 
-	flag.BoolVar(&options.quiet, "verbose", options.quiet, "operate in quiet mode - only emit errors to log")
```

Comment:
verbose is default. `-quiet` will be 'not verbose'.

This was done mostly as a backwards compatibility move where we started with verbose logging and users requested a quieter mode of operation.